@here/harp-mapview/lib/ScreenProjector.ts

```diff
@@ -88,7 +89,10 @@ export class ScreenProjector {
      * @param {THREE.Vector3} target The target vector.
      * @returns {THREE.Vector3} The projected vector (the parameter 'target').
      */
-    projectVector(source: THREE.Vector3 | THREE.Vector4, target: THREE.Vector3): THREE.Vector3 {
+    projectVector(
+        source: THREE.Vector3 | THREE.Vector4 | Vector3Like,
```

Comment:
No need for Vecto3 and Vector4. They are both fulfilling the Vector3Like interface

Reply:
See: 0716eb78a39efe80daee449146d5911b526e8dfb